rag/web_crawler/scraper2.py

```
@@ -0,0 +1,80 @@
+import asyncio
+import time
+
+from crawl4ai import CrawlerRunConfig, AsyncWebCrawler, CacheMode
```

Ruff (0.12.2)

`crawl4ai.CacheMode` imported but unused

Remove unused import: `crawl4ai.CacheMode`

(F401)